app/views/ncr/proposals/new.html.erb

```diff
@@ -2,7 +2,7 @@
   <h4>National Capital Region</h4>
   <h3>Purchase Card Approval Request</h3>
 
-  <%= form_for @proposal_form do |f| %>
+  <%= form_for @proposal_form, url: @form_url, method: @form_method do |f| %>
```

Comment:
Pretty sure the additions aren't needed. From http://guides.rubyonrails.org/form_helpers.html#relying-on-record-identification:

> you can just pass the model instance and have Rails figure out model name and the rest... Record identification is smart enough to figure out if the record is new by asking record.new_record?. It also selects the correct path to submit to and the name based on the class of the object.